java/dagger/android/processor/AndroidProcessor.java

```diff
@@ -53,10 +53,17 @@
 public final class AndroidProcessor extends BasicAnnotationProcessor {
   private static final String FLAG_EXPERIMENTAL_USE_STRING_KEYS =
       "dagger.android.experimentalUseStringKeys";
+  private static final String FLAG_FORMAT_GENERATED_SOURCES =
+      "dagger.formatGeneratedSource";
```

Comment:
I just noticed that this is reading a Dagger flag from Dagger Android. While not the worst thing, it is something I think we want to try to avoid as we try to keep them as separate as possible. I'm thinking of instead just removing the formatting entirely, especially since in Hilt we don't try to format our generated sources at all. Would this satisfy your use case? I assume you want this flag just to turn it off entirely?

Reply:
That works for me!